### ISSUE TITLE
Followers server

### ISSUE THREAD

**Issue by dcadenas** [planetary-social/nos-followers#1]
- [x] Implement a server that calculates the diff between nostr contact lists (kind 3) and detect follows and unfollows.
- [x] Connect to notifications server through a google pubsub
- [x] Ansible playbook
- [x] Consume follows pubsub from Notification server
- [x] Follower server tests
- [x] Switch from test relay to wss://relay.nos.social
- [x] Use settings.production.yml in ansible
- [x] Collpase follow events toggled in rapid succession 
- [x] Follower server grafana dashboard and more metrics
- [x] Add some method to ignore followers from initial follow list never seen before if it's much older than the account creation date (kind 0 can be used for this comparison). It's an heuristic to avoid receiving follow notifications from people that followed you a long time ago
- [x] Create notifications with counters that include the details in the payload for users with too many followers.
- [x] Create a backfill for existent contact lists in the relay. We could ignore most old lists though, we'd just have less notifications from older accounts, see previous item - **Nice to have**, but it would reduce the initial surge of new follower notifications that were in fact following the user a long time ago.
- [x] Check contact lists with no content that have the Nos tag, ensure a grafana alert is triggered. Context [here](https://planetary-app.slack.com/archives/C04PLR5QFQQ/p1725461750304919)
- [x] Move server to production droplet. I ended up pointing followers.nos.social to my dev server, now I need a new dev server!)
- [x] Add notification server changes to deal with the new localized notifications done in https://github.com/planetary-social/nos/pull/1483
- [x] #24
- [x] ~~Split neo4j db into it's own instance (**@gergelypolonkai** has something ready for this)~~
- [x] Read notification metadata so that tapping on it opens the correct profile (**iOS team**)
- [x] Localized notifications as explained [here](https://developer.apple.com/documentation/usernotifications/generating-a-remote-notification) **(iOS team**, at least partially, some changes on the notifications service too)

**Comment by setch-l** [comment]
@dcadenas - As part of this work are you planning to turn off the unfollow Notifications? I'm still getting them. If so do you need a ticket for this or do you want to add it to the check list above?

**Comment by dcadenas** [comment]
Yes, those will be part of my notification server changes item. I won't send unfollowed info although unfollows will properly be reflected in the db and the app will have that info available if it decides to consume it

**Comment by dcadenas** [comment]
@setch-l this is still needing a review for the [ansible script PR](https://github.com/planetary-social/ansible-scripts/pull/119) but we are moving away from it so maybe it's only useful as an example and I don't think the epic should be blocked by that. I disabled the feature flags as it's the only real way to test if the notifications server handles the new events well.

It's already live but a UAT pass will be useful anyways, maybe it's a better idea so have some kind of demo like exploration?
